app/views/AlreadyRegisteredView.scala.html

```diff
@@ -20,7 +20,7 @@
         govukInsetText: GovukInsetText
 )
 
-@(utr: String, formattedAddress: HtmlContent)(implicit request: Request[_], messages: Messages)
+@(utr: String, formattedAddress: HtmlContent, accountLink: String)(implicit request: Request[_], messages: Messages)
```

Comment:
Rather than pass the accountLink it would be better to just have the FrontendAppConfig as an implicit parameter in the view itself.

Reply:
Should be done